geometry_msgs/PointStamped focus_point
---
object_survey/CameraScan scan